test/e2e/framework/framework.go

```diff
@@ -190,6 +192,16 @@ func (f *Framework) BeforeEach() {
 		f.StagingClient, err = staging.NewForConfig(clientRepoConfig)
 		Expect(err).NotTo(HaveOccurred())
 		f.ClientPool = dynamic.NewClientPool(config, api.Registry.RESTMapper(), dynamic.LegacyAPIPathResolverFunc)
+		if ProviderIs("kubemark") && TestContext.ExternalKubeConfig != "" && TestContext.CloudConfig.KubemarkProvider == nil {
+			kubemarkClient, err := clientset.NewForConfig(config)
+			Expect(err).NotTo(HaveOccurred())
+			externalConfig, err := clientcmd.BuildConfigFromFlags("", TestContext.ExternalKubeConfig)
```

Comment:
Please also set `externalConfig.QPS` and `exnternalConfig.Burst` like we do for `config`.